DOCS/man/options.rst

```diff
@@ -4862,7 +4862,7 @@ The following video options are currently all specific to ``--vo=gpu`` and
     auto
         auto-select (default)
     cocoa
-        Cocoa/OS X
+        Cocoa/OS X (deprecated, use --vo=opengl-cb instead)
```

Comment:
Sorry, I missed this previously. I was thinking of adding something under the `vo=opengl-cb` entry which currently says "For use with libmpv direct OpenGL embedding; useless in any other contexts."

Reply:
added something there too. if it's fine i will merge that commit.